spec/descriptions/tagSyntheticResults.md

```diff
@@ -1,6 +1,12 @@
 The endpoints of this group retrieve the results for defined Synthetic tests.
 These endpoints are only available for invited customers for the Synthetic Monitoring Technology Preview.
 
+**Note on name in TagFilter/TagFilterExpression**: From R243, the name used in a TagFilter or a TagFilterExpression has the format: synthetic.\<name\>.
+It can be one of the following: synthetic.id (id is the test result id), synthetic.testId,
+synthetic.testName, synthetic.locationId, synthetic.applicationId, synthetic.serviceId, synthetic.syntheticType,
+synthetic.locationName, and synthetic.locationLabel. If it is a metric name, then the format is: synthetic.metrics\<MetricName\>.
+For example, synthetic.metricsResponseTime, synthetic.metricsStatus.
+
 ## Get Synthetic test playback results 
 The endpoint returns the aggregated Synthetic test result data
 
```

Comment:
I can't seem to a comment in "Mandatory Parameters".  In the **metrics** part maybe add an example at the end of this section about how to specify one metric now.

Reply:
Why is "metrics" appearing in both the "mandatory params" and "optional params"?  Should one spot be "metrics" and the other spot be "metric"?

Reply:
okay, my view must be messed up, I will complete my review and try again.

Reply:
Ignore my deleted comment about payloads... they are correct.

Reply:
We have not changed the metric names, they are still, for example, response_time. The granularity in metrics is an optional parameter, not the whole metrics block.

Reply:
Okay, it wasn't very clear to me which part of `metrics` is required and optional.  I guess the name of the metric is required but granularity is optional.   We can leave it as it is.